fairlearn/reductions/_moments/conditional_selection_rate.py

```diff
@@ -182,3 +219,35 @@ def load_data(self, X, y, **kwargs):
         super().load_data(X, y,
                           event=pd.Series(y).apply(lambda y: _LABEL + "=" + str(y)),
                           **kwargs)
+
+
+class ErrorRateRatio(ConditionalSelectionRate):
+    r"""Implementation of Error Rate Ratio as a moment.
+
+    Measures the ratio in errors per attribute by overall error.
+    The 2-sided version of error ratio can be written as
+    ratio <= error(A=a) / total_error <= 1/ratio
+    .. math::
+    ratio <= E[abs(h(x) - y)| A = a] / E[abs(h(x) - y)] <= 1/ratio\; \forall a
+
+    This implementation of :class:`ConditionalSelectionRate` defines a single event, `all`.
+    Consequently, the `prob_event` :class:`pandas:pandas.DataFrame` will only have a single
```

Comment:
Drop the class designation or fix to `pandas.Series`

Reply:
Fixed to `pandas.Series`